src/_fade.scss

```diff
@@ -28,17 +39,29 @@
 	animation-fill-mode:forwards;
 }
 
+.anim-fade--out--short {
+	animation:fade--out $duration-s ease-out;
+	animation-iteration-count:1;
+	animation-fill-mode:forwards;
+}
+
+/*
+	transitions for js, in response to events and user interaction
+*/
 .anim-opacity--0 {
-    opacity:0;
+	opacity:0;
 }
 
 .anim-opacity--1 {
-    opacity:1;
+	opacity:1;
 }
 
-// for popover example
-.anim-opacity--short {
-    transition: opacity $duration-s ease-in-out;
+.anim-transition--opacity--s {
+	transition: opacity $duration-s ease-in-out;
+}
+
+.anim-transition--opacity {
+	transition: opacity $duration-m ease-in-out;
 }
 
```

Comment:
animations named so they can play well with react can be added once we have a need for things to appear when items are added or removed etc.